ext/SparseMatrixColoringsColorTypesExt.jl

```diff
@@ -0,0 +1,102 @@
+# Visualize colored matrices using the Julia Images ecosystem.
+# ColorTypes.jl is the most light-weight dependency to achieve this. 
+module SparseMatrixColoringsColorTypesExt
+
+using SparseMatrixColorings: SparseMatrixColorings
+using SparseMatrixColorings: AbstractColoringResult, ColumnColoringResult, RowColoringResult
+using ColorTypes: Colorant, RGB, RGBA
+
+# Default to Makie.jl's default color scheme "Wong":
+# https://github.com/MakieOrg/Makie.jl/blob/e90c042d16b461e67b750e5ce53790e732281dba/src/theming.jl#L1-L16
+# Conservative 7-color palette from Points of view: Color blindness, Bang Wong - Nature Methods
+# https://www.nature.com/articles/nmeth.1618?WT.ec_id=NMETH-201106
+DEFAULT_COLOR_SCHEME = [
+    RGB(0 / 255, 114 / 255, 178 / 255),   # blue
+    RGB(230 / 255, 159 / 255, 0 / 255),   # orange
+    RGB(0 / 255, 158 / 255, 115 / 255),   # green
+    RGB(204 / 255, 121 / 255, 167 / 255), # reddish purple
+    RGB(86 / 255, 180 / 255, 233 / 255),  # sky blue
+    RGB(213 / 255, 94 / 255, 0 / 255),    # vermillion
+    RGB(240 / 255, 228 / 255, 66 / 255),  # yellow
+]
+DEFAULT_BACKGROUND = RGBA(0, 0, 0, 0)
+const DEFAULT_SCALE = 1   # update docstring in src/images.jl when changing this default
+const DEFAULT_PADDING = 0 # update docstring in src/images.jl when changing this default
+
+ncolors(res::AbstractColoringResult) = maximum(res.color)
```

Comment:
```suggestion
ncolors(res::AbstractColoringResult{s,:column}) where {s} = maximum(column_colors(res))
ncolors(res::AbstractColoringResult{s,:row}) where {s} = maximum(row_colors(res))
```
You can apply the same logic below to show either the column or row colors in every case.